### ISSUE TITLE
Interpreting the loss values

### ISSUE THREAD

**Issue by dalonlobo** [google/uis-rnn#12]
Hi Team,

I'm training the model on a custom dataset. I'm finding it confusing to interpret the various losses displayed during training. For example, does having large negative Training loss is better or should I concentrate on Negative log likelihood? To sum up, how will I know that the model is converging? Following image is from my training process.

![image](https://user-images.githubusercontent.com/12654849/50162871-50649680-0305-11e9-9559-3d743e1917a9.png)

Your help is greatly appreciated.

**Comment by wq2012** [comment]
You could plot the training loss with matplotlib to see whether it's converging or not. We didn't add it by default because we don't want to have too many dependencies.

But in general, smaller value of loss means better model.
